### PR TITLE
Support BlockPCs in Alive backend

### PR DESCRIPTION
**Pull request by manasij7479**
Do not merge until alive supports phi

### CODE REVIEW

File: lib/Infer/ExhaustiveSynthesis.cpp [hunk 1]
```diff
@@ -57,7 +57,7 @@ namespace {
     cl::init(false));
   static cl::opt<bool, /*ExternalStorage=*/true>
     AliveFlagParser("souper-use-alive", cl::desc("Use Alive2 as the backend"),
-    cl::Hidden, cl::location(UseAlive), cl::init(false));
+    cl::Hidden, cl::location(UseAlive), cl::init(true));
```
Comment (regehr) [inline lib/Infer/ExhaustiveSynthesis.cpp]: nor yet!!!!

**Comment by regehr** [comment]
this is great! 
I guess it's impossible to test until alive2 has phi?

**Comment by regehr** [comment]
this is still open from February 20, please finish it up

**Comment by manasij7479** [comment]
This is probably best closed.

**Pull request closed without merging**